src/server/editorServices.ts

```diff
@@ -1966,6 +1966,11 @@ namespace ts.server {
             return configProject && configProject.getCompilerOptions().configFile;
         }
 
+        /* @internal */
+        allScriptInfoFileNamesForDebug(): ReadonlyArray<string> {
+            return arrayFrom(this.filenameToScriptInfo.keys());

```

Comment:
I think we should have both filename and path print instead of just path to figure out if there is mismatch in casing that could cause this.

Reply:
So the keys of `filenameToScriptInfo` are paths and not filenames? I was assuming the key to this map was the same as the value's `.fileName` property?

Reply:
No it is path.